crates/bevy_audio/src/sinks.rs

```diff
@@ -85,7 +85,7 @@ pub trait AudioSinkPlayback {
 /// that source is unchanged, that translates to the audio restarting.
 #[derive(Component)]
 pub struct AudioSink {
-    pub(crate) sink: Sink,
+    pub sink: Sink,
```

Comment:
Doc strings for these fields now that they're pub.

Reply:
Ya sorry this should be draft still